docs/UserGuide.md

```diff
@@ -37,15 +37,15 @@ NUSpam is a desktop app for managing contacts **targeted at marketers who requir
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-   - **`list`** : Lists all contacts.
+   - `list` : Lists all contacts.
 
-   - **`add`**`-n “John Doe” -p “+659875432” -e “johnd@example.com”` : Adds a contact named `John Doe` to the Address Book.
```

Comment:
Agree with you, primarily we want these inline code to be attached. With that we cannot bold selective words in the code without html tags, which we should avoid the overuse of.